### PR TITLE
Fixing band offsets and output directory, removing some products

### PR DESCRIPTION
**Pull request by mnicholl**

### CODE REVIEW

File: mosfit/models/magnetar/magnetar.json [hunk 1]
```diff
@@ -26,33 +26,6 @@
             }
         }
     },
-    "ebv":{
```
Comment (guillochon) [inline mosfit/models/magnetar/magnetar.json]: So these lines are in the JSON file to facilitate running the code in a "parameter scan" mode where they might be randomly varied. Any reason these were deleted in this PR, were they breaking something?

File: mosfit/models/magnetar/magnetar.json [hunk 2]
```diff
@@ -149,37 +117,34 @@
...
         "requests":[
             ["bandwavelengths", "filters"],
-            [],
             []
         ]
     },
     "filters":{
         "kind":"observable",
         "inputs":[
-            "extinction",
```
Comment (guillochon) [inline mosfit/models/magnetar/magnetar.json]: This is related to above, the `Filter` class needs to know that `lumdist` is a parameter that can be passed, so these lines should remain unless it was breaking things (in which case we should fix the breakage!)

**Comment by mnicholl** [comment]
No reason, this file shouldn’t have been included as part of the PR either. I was just experimenting with parameters, but did an ‘add all’ after fixing bugs in those other files. Still getting the hang of this git business. Can you merge only the filterrules.json and mosfit.ipynb files?

**Comment by guillochon** [comment]
So if you push any further commits to your fork to GitHub, this pull request will automatically be updated. You don't have to close and submit a new one.

**Comment by guillochon** [comment]
Do you have a slsn JSON file for your model? I'd suggest making a new model folder with both a slsn.json and parameters.json for the PR. They'd almost be clones of magnetar but changing envelopecore to slsn.

**Comment by guillochon** [comment]
Alrighty looks good! I will merge and add a test for Travis.

**Pull request merged**